docs/commands/mv.md

```diff
@@ -0,0 +1,74 @@
+# mv
+
+Move storage files and directories through Studio.
+
+## Synopsis
+
+```usage
+usage: datachain mv [-h] [-v] [-q] [--recursive]
+     [--team TEAM] [-s] path new_path
+```
+
+## Description
+
+This command moves files and directories within storage. The move operation is performed within the same bucket - you cannot move files between different buckets. The command supports both individual files and directories, with the `--recursive` flag required for moving directories.
+
+## Arguments
+
+* `path` - Path to the storage file or directory to move
+* `new_path` - New path where the file or directory should be moved to
+
+## Options
+
+* `--recursive` - Move recursively
```

Comment:
is it a regular thing to do require `--recursive` for mv?

Reply:
I am not sure, will remove it.